src/java/generator/MapGenerator.java

```diff
@@ -20,34 +23,52 @@
 	public static final String VERSION = "0.1.0";
 
 	public static void main(String[] args) throws ExecutionException, InterruptedException {
+
+		String folderPath ="";
+		String version ="";
+		String mapName ="";
+		long seed = 0L;
+
 		try {
-			String folderPath = args[0];
-			long seed = Long.parseLong(args[1]);
-			String version = args[2];
-			String mapName = args.length >= 4 ? args[3] : "NeroxisGen_" + VERSION + "_" + seed;
-			
+			folderPath = args[0];
+			seed = Long.parseLong(args[1]);
+			version = args[2];
+			mapName = args.length >= 4 ? args[3] : "NeroxisGen_" + VERSION + "_" + seed;
```

Comment:
that looks terrible, that should really be a String.format, not your fault though, ignore it for now